src/pydicom/cli/main.py

```diff
@@ -124,6 +126,9 @@ def filespec_parser(filespec: str) -> list[tuple[Dataset, Any]]:
     """
     prefix, filename, element = filespec_parts(filespec)
 
+    # filename = Path(filename)
+    # if filename.is
+
```

Comment:
Is this a leftover?

Reply:
Yes, started code to check for absolute path and thought of different way ... will remove.